test/src/api/commands/element/testClick.js

```diff
@@ -188,4 +188,50 @@ describe('.click()', function() {
       client.start(done);
     });
   });
+
+  it('client.click() - for native app', function(done) {
+    Nightwatch.initW3CClient({
+      output: false,
+      silent: false,
+      selenium: {
+        start_process: false,
+        port: 10195,
+        host: 'localhost'
+      },
+      desiredCapabilities: {
+        'appium:automationName': 'XCUITest',
+        browserName: null,
+        'appium:app': '../samples/Wikipedia.app',
+        'appium:appPackage': 'org.wikimedia.wikipedia',
+        platformName: 'iOS',
+        'appium:deviceName': 'iPhone 13',
+        'appium:platformVersion': '15.5'
+      }
+    }).then(client => {
+      MockServer.addMock({
+        'url': '/wd/hub/session/13521-10219-202/elements',
+        'postdata': {'using':'css selector','value':'#webdriver'},
+        'response': {
+          value: [{
+            'element-6066-11e4-a52e-4f735466cecf': '5cc459b8-36a8-3042-8b4a-258883ea642b'
+          }],
+          status: 0
+        }
```

Comment:
We already have a mock for this in `mocks-w3c.yml` so I think this mihgt not be needed